src/MICore/LaunchOptions.cs

```diff
@@ -172,6 +172,39 @@ public EnvironmentEntry(Xml.LaunchOptions.EnvironmentEntry xmlEntry)
         public string Value { get; private set; }
     }
 
+    public sealed class PathMapEntry
+    {
+        public PathMapEntry()
+        {
+        }
+
+        public PathMapEntry(Xml.LaunchOptions.PathMapEntry xmlEntry)
+        {
+            if (string.IsNullOrEmpty(xmlEntry.EditorPath))
+            {
+                throw new ArgumentNullException("EditorPath");
+            }
+            this.EditorPath = xmlEntry.EditorPath;
+            this.CompilerPath = xmlEntry.CompilerPath == null ? string.Empty : xmlEntry.CompilerPath;
+            this.UseForBreakpoints = xmlEntry.UseForBreakpoints;
+        }
+
+        public string EditorPath { get; set; }
```

Comment:
If the two cases are CompileTime and Current I think that these are obvious. I could make it even more explicit, e.g. "CurrentLocal", if you thought it was better. Sure, ask.